base/shared/src/main/scala/scalaz/std/option.scala

```diff
@@ -0,0 +1,15 @@
+package scalaz
+package std
+
+import scala.{ Option, Some }
+
+import scalaz.ct._
+
+trait OptionInstances {
```

Comment:
we could, separately, use with some more of these.

Reply:
Apparently, yes. Some of the instances were duplicated. I just copy/pasted all the instances I deleted (or most of them. I think I missed a couple. Oops.). I guess I'll move everything to the std module then...